tool/gravity/cli/run.go

```diff
@@ -58,6 +59,20 @@ func ConfigureEnvironment() error {
 // Run parses CLI arguments and executes an appropriate gravity command
 func Run(g *Application) error {
 	log.Debugf("Executing: %v.", os.Args)
+	if runErr := run(g); runErr != nil {
+		msg := fmt.Sprintf("Failed to exec [%s]: %s", strings.Join(os.Args, " "), trace.UserMessage(runErr))
+		if logErr := utils.SyslogWrite(syslog.LOG_INFO, msg, constants.GravityCLITag); logErr != nil {
+			log.WithError(logErr).Warn("Failed to write to system logs.")
+		}
+		return runErr
+	}
+	if err := utils.SyslogWrite(syslog.LOG_INFO, strings.Join(os.Args, " "), constants.GravityCLITag); err != nil {
```

Comment:
Hmm, so this will now only write the command in the logs _after_ it has been executed. This means that history will potentially miss long-running commands that for example were started by someone and then Ctrl-C'd. I would just log the command first thing as soon as "run" is invoked. Didn't you implement it this way before?

Reply:
@bernardjkim We can log it twice - first when the command is just being launched, and then once it's finished - along with the result (success/error) to include Dima's suggestion.

Reply:
@bernardjkim And even better - also generate a uuid4 for each command invocation and log alongside "started/finished" log entries - this way it will be easy to trace each particular command invocation in these logs (e.g. find where one "gravity status" call started/finished and distinguish it from another).